src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -257,25 +259,29 @@ const ReservationList: FC<ReservationListProps> = ({ pageSize }) => {
 
   return (
     <>
-      <div className={`reservation-list-page ${getScrollClass(modalIds)}`}>
-        <h1 className="text-header-h1 m-32">
-          {t("reservationListHeaderText")}
-        </h1>
-        {user && <ReservationPauseToggler user={user} />}
-        {allListsEmpty && <EmptyReservations />}
-        {!allListsEmpty && (
-          <DisplayedReservations
-            openReservationDetailsModal={openReservationDetailsModal}
-            readyForPickupReservationsFBS={readyForPickupReservationsFBS}
-            readyForPickupReservationsPublizon={
-              readyForPickupReservationsPublizon
-            }
-            reservedReservationsFBS={reservedReservationsFBS}
-            reservedReservationsPublizon={reservedReservationsPublizon}
-            pageSize={pageSize}
-          />
-        )}
-      </div>
+      {modalIds.length === 0 && (
```

Comment:
I think this should be reverted: <div className={`reservation-list-page ${getScrollClass(modalIds)}`}>